e2e/tests/test_e2e.py

```diff
@@ -212,6 +220,90 @@ def test_logical_backup_cron_job(self):
         self.assertEqual(0, len(jobs),
                          "Expected 0 logical backup jobs, found {}".format(len(jobs)))
 
+    # zz --> Other tests depend on the state of the regular acid-minimal-cluster, which this test deletes, so run last.
+    @timeout_decorator.timeout(TEST_TIMEOUT_SEC)
+    def test_zz_node_selector(self):
+        k8s = self.k8s
+
+        # Delete the default cluster, but pause first so delete works properly
+        time.sleep(60)
+        default_cluster_label = 'version=acid-minimal-cluster'
+        print ("Deleting default test cluster")
+        k8s.delete_clusters("acid-minimal-cluster")
+        k8s.wait_for_pods_deleted(default_cluster_label)
+
+        # 1. Add a label to a node
+        target_node = k8s.api.core_v1.read_node(name='postgres-operator-e2e-tests-worker2')
+        pg_patch_env_special = {
+            "metadata": {
+                "labels": {
+                    "environment": "special"
+                }
+            }
+        }
+        target_node_name = target_node.metadata.name
+        print ("Patching node [%s] with 'environment:special' label" %(target_node_name))
```

Comment:
my flake 8 linter tells me, it doesn't like the space after print. Put it after the % operator and it's happy:

```suggestion
        print("Patching node [%s] with 'environment:special' label" % (target_node_name))
```